tests/sentry/issues/test_escalating.py

```diff
@@ -259,7 +259,7 @@ def test_datetime_number_of_days() -> None:
 
 class DailyGroupCountsEscalating(BaseGroupCounts):
     def save_mock_escalating_group_forecast(
-        self, group: Group, forecast_values=list[int], date_added=datetime
+        self, group: Group, forecast_values: list[int], date_added: datetime
```

Comment:
This is the only non-typing change, no actual effect since defaults are never used and they clearly would have been broken if they were.